src/main/java/baritone/utils/PathRenderer.java

```diff
@@ -127,12 +129,12 @@ private static void drawPath(IPath path, int startIndex, Color color, boolean fa
             int dirY = end.y - start.y;
             int dirZ = end.z - start.z;
 
-            while (next + 1 < positions.size() && (!fadeOut || next + 1 < fadeStart) &&
+            /*while (next + 1 < positions.size() && (!fadeOut || next + 1 < fadeStart) &&
                     (dirX == positions.get(next + 1).x - end.x &&
                             dirY == positions.get(next + 1).y - end.y &&
                             dirZ == positions.get(next + 1).z - end.z)) {
                 end = positions.get(++next);
-            }
+            }*/
```

Comment:
this can probably be added back, or made conditional. i only commented it out so that the elytra segments would be clearly visible.